.gitignore

```diff
@@ -0,0 +1,58 @@
+# Logs
```

Comment:
minor nitpick: could you keep .gitignore out of this commit? I stick to using a global .gitignore file, so that we don't encounter editor-specific sprawl - otherwise everyone would have to put up with my `*.swp` rules :)